rs/sns/swap/proto/ic_sns_swap/pb/v1/swap.proto

```diff
@@ -221,8 +221,9 @@ message Swap {
   // to the outcome of the swap.
   repeated SnsNeuronRecipe neuron_recipes = 7;
 
-  // Gets set to whatever value is in the corresponding field of OpenRequest
-  // (that field is required at the application level).
+  // The proposal ID that was used to create the SNS that opened this swap.
+  // Note: the name is a historical artifact because the swap used to be opened
+  // with an OpenSnsTokenSwap request.
```

Comment:
This gets set during initial code installation, right? Doesn't that mean it comes from Init? These facts seem worth mentioning. Could this also be deprecated?

Reply:
> This gets set during initial code installation, right? Doesn't that mean it comes from Init? 

Yes I believe it does

> These facts seem worth mentioning.

I agree, I will add it

> This gets set during initial code installation, right? Doesn't that mean it comes from Init? These facts seem worth mentioning. Could this also be deprecated?

See my reasoning in [this comment](https://github.com/dfinity/ic/pull/454#discussion_r1684601822) for why I didn't deprecate it